modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,355 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups create a cron job that starts a single backup each time it triggers.
+
+:FeatureName: Automating etcd backups
+include::snippets/technology-preview.adoc[]
+
+[id="enabling-automated-etcd-backups_{context}"]
+== Enabling automated etcd backups
+
+Follow these steps to enable the automated backups feature for etcd.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `FeatureGate` custom resource (CR) file named `etcd-enable-automated-backups.yaml` with the following contents:
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
```

Comment:
```suggestion
When the `TechPreviewNoUpgrade` feature set is enabled on your cluster, it prevents minor version updates. The `TechPreviewNoUpgrade` feature set cannot be disabled. Do not enable this feature set on production clusters.
```
- I divided the two main points from the first sentence into two sentences, as they each seem important enough to stand alone.
- Per IBM Style [Word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage), whenever possible, rewrite statements that include "should" to be more direct.
- Also, given the importance of this warning, I wonder if it should be placed before the procedure? I understand why you placed it near the step where you add the feature set, but I think I'd want to know the information in the warning before I started reading the procedure.

Reply:
All good points, thank you!